Article/Article.js

```diff
@@ -26,4 +33,23 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll('.article');
+articles.forEach(article => {
+  return new Article(article);
+});
+
+// EXTRAS:
+const articleHeaders = document.querySelector('.article h2');
+articleHeaders.addEventListener('mouseover', (e) => {
```

Comment:
This section of code seems to have broken your project. When I commented it out, your page ran fine. I hope you can get this working, I wanted to see the animation.

Check your querySelector on line 42.